### ISSUE TITLE
Creat histogram style filter component

### ISSUE THREAD

**Issue by alexluscombe** [sajari/sajari-sdk-react#98]
Create a histogram style filter component. e.g. see price filtering component on Airbnb search results page. Component should include all the functionality from the AIrbnb version.

![image](https://user-images.githubusercontent.com/35442403/50427330-2872c580-08f9-11e9-9a3d-dee3da61318f.png)

Component should allow the user to see the distribution of values of a particular field and allow them to narrow results by dragging the  handles at the upper and lower ends of the slider.  Slider handles should not be allowed to cross over one and other.

As the handles are dragged along the slider the user should be able to see how they are changing the filter values. A Clear button should reset the component to its original state. An apply button should run the search with the new parameters set.

**Comment by zlatanpham** [comment]
@mish15 you meant color of the highlighted area on the attachment could be changed? 
<img width="459" alt="screen shot 2019-01-09 at 9 35 21 am" src="https://user-images.githubusercontent.com/12707960/50872410-32fd8680-13f2-11e9-88e0-a732b66ff7b6.png">
For the mobile drag support, no worries, I did complete the feature yesterday.

**Comment by mish15** [comment]
@zlatanpham yes exactly. I would guess normally the selected area would be coloured and the areas outside would be greyed out. Probably useful to be able to set both though

**Comment by zlatanpham** [comment]
@mish15 👍 I'll post the demo once completed.

**Comment by zlatanpham** [comment]
@mish15 [here](https://codesandbox.io/s/6wwjron2ok) is the update.

**Comment by mish15** [comment]
Cool perfect!

**Comment by alexluscombe** [comment]
Hi @zlatanpham We're happy with the prototype. Can you create a PR for @benhinchley  to review?

**Comment by zlatanpham** [comment]
Okay
